operator/src/test/resources/expected/custom-config-strimzi.yml

```
@@ -121,6 +121,12 @@ spec:
       leader.imbalance.per.broker.percentage: 0
       connections.max.reauth.ms: 299000
       create.topic.policy.class.name: io.bf2.kafka.topic.ManagedKafkaCreateTopicPolicy
+      strimzi.authorization.custom-authorizer.partition-counter.max.partitions: 3000
+      strimzi.authorization.custom-authorizer.partition-counter.timeout-seconds: 10
+      strimzi.authorization.custom-authorizer.partition-counter.private-prefix: __redhat_
```

```suggestion
      strimzi.authorization.custom-authorizer.partition-counter.private-topic-prefix: __redhat_
```
See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/678/files#r822214537 for more discussion

---

Done, thanks!